recreate_v4_files_listing.py

```diff
@@ -20,7 +22,7 @@ def create_v4_images_listfile(search_dirpath=V4_DIR, filespec='*.png',
                               output_filepath=V4_LISTFILE_PATH):
     file_names = get_v4_imagefile_names(search_dirpath, filespec)
     with open(output_filepath, 'w') as f_out:
-        for file_name in file_names:
+        for file_name in sorted(file_names):
```

Comment:
Sorting ensures that a newly generated `v4_files_listing.txt` file is ordered, making PR changes easier to review - otherwise it's not guaranteed that the same `v4_files_listing.txt` can be recreated.

Reply:
This will make things much easier!